README.md

```diff
@@ -138,12 +134,34 @@ _Additional information_
 * `field_delimiter` (`default=none`)
   * Custom field delimiter, for when format is set to `TEXTFILE`
 * `table_properties`: table properties to add to the table, valid for Iceberg only
-* `lf_tags` (`default=none`)
-  * lf tags to associate with the table
-  * format: `{"tag1": "value1", "tag2": "value2"}`
-* `lf_tags_columns` (`default=none`)
-  * lf tags to associate with the table columns
-  * format: `{"tag1": {"value1": ["column1": "column2"]}}`
+* `lf_tags_config` (`default=none`)
+  * [AWS lakeformation](#aws-lakeformation-integration) tags to associate with the table and columns
+  * format for model config:
+```json
+{
+  "enabled": true,
+  "tags": {
+    "tag1": "value1",
+    "tag2": "value2"
+  },
+  "tags_columns": {
+    "tag1": {
+      "value1": ["column1", "column2"]
+    }
+  }
+}
+```
+* format for `dbt_project.yml`:
+```yaml
+  +lf_tags_config:
+    enabled: true
+    tags:
+      tag1: value1
+      tag2: value2
+    tags_columns:
+      tag1:
+        value1: [ column1, column2 ]
```

Comment:
should be `value` instead of `value1 `? Also in term of data structure why do you use value?
can we use something like that:
```
tags_columns:
      tag1:[ column1, column2 ] 
```
I find the usage of value a bit redundant.

Reply:
It can look like the following and it's not redundant:
```
   tags_columns:
      tag1:
        value1: [ column1, column2 ]
        value2: [ column3, column4 ]
```

Reply:
ah jez, I got it. For a specific tag we also need to pass the value indeed, my bad.